src/game/client/components/items.cpp

```diff
@@ -284,15 +375,8 @@ void CItems::OnRender()
 		{
 			RenderLaser((const CNetObj_Laser *)pData);
 		}
-	}
-
-	// render flag
-	for(int i = 0; i < Num; i++)
-	{
-		IClient::CSnapItem Item;
-		const void *pData = Client()->SnapGetItem(IClient::SNAP_CURRENT, i, &Item);
-
-		if(Item.m_Type == NETOBJTYPE_FLAG)
+		// render flags
+		else if(Item.m_Type == NETOBJTYPE_FLAG)
```

Comment:
I think this changes the render order of items. Previously flags were always in front of the other items, which seems intentional.

Reply:
Yeah you are right! I wondered why the loops aren't put into each other!